tmail-backend/tmail-third-party/ai-bot/src/main/java/com/linagora/tmail/mailet/AIBotMailet.java

```diff
@@ -154,26 +154,24 @@ private MimeMessage evaluateReplyMimeMessage(Mail mail, String answer) throws Me
 
     private void stripBotFromRecipients(MimeMessage reply, Message.RecipientType recipientType) throws MessagingException {
         Address[] toAddresses =
-            Arrays.stream(Optional.ofNullable(reply.getRecipients(recipientType))
-                    .orElse(new InternetAddress[0]))
-                .filter(Throwing.predicate(address -> !address.equals(
-                    new InternetAddress(config.getBotAddress().asString()))))
-                .toArray(Address[]::new);
+                Arrays.stream(Optional.ofNullable(reply.getRecipients(recipientType))
+                                .orElse(new InternetAddress[0]))
+                        .filter(Throwing.predicate(address -> !address.equals(
+                                new InternetAddress(config.getBotAddress().asString()))))
+                        .toArray(Address[]::new);
         reply.setRecipients(recipientType, toAddresses);
     }
 
     private List<MailAddress> evaluateRecipients(Mail mail) {
         return Stream.of(mail.getMaybeSender().asList(), mail.getRecipients())
-            .flatMap(Collection::stream)
-            .filter(recipient -> !recipient.equals(config.getBotAddress()))
-            .toList();
+                .flatMap(Collection::stream)
+                .filter(recipient -> !recipient.equals(config.getBotAddress()))
+                .toList();
     }
 
     @Override
     public void init() throws MailetException {
-        this.config = AIBotConfig.fromMailetConfig(getMailetConfig());
         this.chatLanguageModel = createChatLanguageModelModel(config);
-
         this.defaultMessageBuilder = new DefaultMessageBuilder();
         defaultMessageBuilder.setMimeEntityConfig(MimeConfig.PERMISSIVE);
         defaultMessageBuilder.setDecodeMonitor(DecodeMonitor.SILENT);
```

Comment:
Not your code but the message builder should be instanciated on a per request basis.

To be addressed in a separate commit.

WILL cause thread safety issues...